docs/_data/bearer_scan.yaml

```diff
@@ -13,6 +13,9 @@ options:
     - name: debug
       default_value: "false"
       usage: Enable debug logs
+    - name: disable-default-rules
+      default_value: "false"
+      usage: Disables all default and built-in rules.
```

Comment:
Ping @markmichon for the new flag and flag description 🙏 
I'm not sure how much detail we want in the description, but for context when enabled, this flag means a Bearer scan would only run rules in the external rules folder.

Reply:
Let's keep it minimal like you have it now. I can enhance the custom rule guide to better explain it.